demo/campaign/src/common/helpers/imageBlockHelpers.ts

```diff
@@ -0,0 +1,23 @@
+export const validSizes: number[] = (process.env.DAM_ALLOWED_IMAGE_SIZES as string)
+    .split(",")
+    .map((value) => parseInt(value))
+    .sort((a, b) => a - b);
+
+export const getDamAllowedImageWidth = (minimumWidth: number, contentWidth): number => {
+    let width: number | null = null;
+    const largestPossibleWidth = validSizes[validSizes.length - 1];
+
+    validSizes.forEach((validWidth) => {
+        if (minimumWidth === contentWidth) {
+            width = 1020;
```

Comment:
Shouldn't this be exactly twice the `contentWidth`? 
The `contentWidth` is exactly the maximum width the image can be rendered in, correct?